Source/WebCore/accessibility/AXObjectCache.cpp

```diff
@@ -4440,7 +4453,22 @@ AXTextChange AXObjectCache::textChangeForEditType(AXTextEditType type)
     return AXTextInserted;
 }
 #endif
-    
+
+AccessibilityObject* AXObjectCache::textCompositionObjectForNode(Node& node)
+{
+    // Find the ax object that sends text change notifications for the composition
+    auto* object = get(&node);
+    if (object)
+        object = object->observableObject();
+
+#if PLATFORM(MAC)
+    // Fallback on the webarea
```

Comment:
This comment seems redundant as-is. Maybe it would be good to explain why we only do this for macOS though? If I recall correctly, it's to match behavior elsewhere, so maybe we can call that out.

Reply:
added some explanation of how we fall back on the Mac for text notifications. It happens in a couple of methods, so I didn't call out any specific one.